src/main/webapp/js/userMapData.json

```diff
@@ -1,12 +1,9 @@
 {
-  "lastUpdated": "15 March 2017",
+  "lastUpdated": "28 February 2017",
   "institutes": {
-  	 "Abu Dhabi": [ 	
-  	   "Khalifa University"
```

Comment:
I see! After searching I think [Khalifa University](http://www.kustar.ac.ae/) is distinct from [Abu Dhabi University](http://www.adu.ac.ae/en-us/home.aspx) though, even though both are in Abu Dhabi, UAE. Maybe add Khalifa University under the UAE section?